### ISSUE TITLE
Using LIFO queue instead of fifo fails

### ISSUE THREAD

**Issue by scientes** [provinzio/CoinTaxman#36]
```
  File "/home/bastian/Documents/git/CoinTaxman/src/main.py", line 44, in <module>
    main()
  File "/home/bastian/Documents/git/CoinTaxman/src/main.py", line 38, in main
    taxman.evaluate_taxation()
  File "/home/bastian/Documents/git/CoinTaxman/src/taxman.py", line 182, in evaluate_taxation
    self.__evaluate_taxation(coin, operations)
  File "/home/bastian/Documents/git/CoinTaxman/src/taxman.py", line 70, in _evaluate_taxation_GERMANY
    balance.remove_fee(op.change)
  File "/home/bastian/Documents/git/CoinTaxman/src/balance_queue.py", line 92, in remove_fee
    self.buffer_fee.append(fee)
AttributeError: 'BalanceLIFOQueue' object has no attribute 'buffer_fee'
```

**Comment by provinzio** [comment]
should be fixed with 6c94956293438d861a9cb699298e76a4f505dd3f. Would you check if your test case does work, too?

**Comment by scientes** [comment]
now it works thanks!
